lib/fuse.c

```diff
@@ -3691,11 +3697,12 @@ static int readdir_fill_from_list(fuse_req_t req, struct fuse_dh *dh,
 
 		if (flags & FUSE_READDIR_PLUS) {
 			struct fuse_entry_param e = {
-				.ino = 0,
+				.ino = de->ino,
```

Comment:
I will look into this patch immediately in the morning, but I'm confused about setting .ino here. `do_lookup`  will overwrite it with `e->ino = node->nodeid;`?

Reply:
Ah sorry, in the lines below.